stacks/spica/src/bucket/pages/index/index.component.ts

```diff
@@ -115,19 +115,23 @@ export class IndexComponent implements OnInit {
 
         setTimeout(() => {
           const propertyNameFirst =
-            this.properties.length > 1 && this.properties[1].name
-              ? this.properties[1].name
-              : undefined;
+            this.properties.length > 0 ? this.properties[1].name : undefined;
```

Comment:
The condition should be `this.properties.length >= 1`

Reply:
I don't need equal operator.

Reply:
Only the first feature is assigned here.

Reply:
There may not be more than one property. In that case, this check will fail as expected cause `this.properties.length` will be equal to 1 and that single property will be stored at `this.properties[0]`.

So when you try to read the item at index 1, it will fail